tests/unit/test_config.py

```
@@ -225,8 +227,165 @@ class TestSettings(Settings):
         )
 
     # 4. 初期化 (.env が優先されるはず)
-    # デフォルトの検証エラーを避けるため、必要なフィールドを指定
-    settings = TestSettings(openai_api_key="sk-test")
+    # デフォルトの検証エラーを避けるため、必要な認証情報のみを指定
+    # 注: default_settings は引数にリストするだけで、すべての環境変数をクリアする効果がある
+    settings = TestSettings(
+        openai_api_key="sk-test",
+        postgres_password="test",
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to argument: "postgres_password"

(S106)